packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -138,12 +156,28 @@ export class AtomicNumericFacet
     );
   }
 
+  private compareRanges(
```

Comment:
Could exporting a `compareRanges` utility be useful for Headless?

Reply:
maybe rename to `areRangesEqual` to be more descriptive of the function's purpose.